### PR TITLE
EO-49466: TF code to create VPC

### PR DESCRIPTION
**Pull request by DivyaMatt**
EO-49466: TF code to create VPC

```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.vpc.aws_default_network_acl.this[0] will be created
  + resource "aws_default_network_acl" "this" {
      + arn                    = (known after apply)
      + default_network_acl_id = (known after apply)
      + id                     = (known after apply)
      + owner_id               = (known after apply)
      + tags                   = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-default"
        }
      + tags_all               = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-default"
        }
      + vpc_id                 = (known after apply)

      + egress {
          + action          = "allow"
          + from_port       = 0
          + ipv6_cidr_block = "::/0"
          + protocol        = "-1"
          + rule_no         = 101
          + to_port         = 0
        }
      + egress {
          + action     = "allow"
          + cidr_block = "0.0.0.0/0"
          + from_port  = 0
          + protocol   = "-1"
          + rule_no    = 100
          + to_port    = 0
        }

      + ingress {
          + action          = "allow"
          + from_port       = 0
          + ipv6_cidr_block = "::/0"
          + protocol        = "-1"
          + rule_no         = 101
          + to_port         = 0
        }
      + ingress {
          + action     = "allow"
          + cidr_block = "0.0.0.0/0"
          + from_port  = 0
          + protocol   = "-1"
          + rule_no    = 100
          + to_port    = 0
        }
    }

  # module.vpc.aws_default_route_table.default[0] will be created
  + resource "aws_default_route_table" "default" {
      + arn                    = (known after apply)
      + default_route_table_id = (known after apply)
      + id                     = (known after apply)
      + owner_id               = (known after apply)
      + route                  = (known after apply)
      + tags                   = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-default"
        }
      + tags_all               = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-default"
        }
      + vpc_id                 = (known after apply)

      + timeouts {
          + create = "5m"
          + update = "5m"
        }
    }

  # module.vpc.aws_default_security_group.this[0] will be created
  + resource "aws_default_security_group" "this" {
      + arn                    = (known after apply)
      + description            = (known after apply)
      + egress                 = (known after apply)
      + id                     = (known after apply)
      + ingress                = (known after apply)
      + name                   = (known after apply)
      + name_prefix            = (known after apply)
      + owner_id               = (known after apply)
      + revoke_rules_on_delete = false
      + tags                   = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-default"
        }
      + tags_all               = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-default"
        }
      + vpc_id                 = (known after apply)
    }

  # module.vpc.aws_eip.nat[0] will be created
  + resource "aws_eip" "nat" {
      + allocation_id        = (known after apply)
      + association_id       = (known after apply)
      + carrier_ip           = (known after apply)
      + customer_owned_ip    = (known after apply)
      + domain               = (known after apply)
      + id                   = (known after apply)
      + instance             = (known after apply)
      + network_border_group = (known after apply)
      + network_interface    = (known after apply)
      + private_dns          = (known after apply)
      + private_ip           = (known after apply)
      + public_dns           = (known after apply)
      + public_ip            = (known after apply)
      + public_ipv4_pool     = (known after apply)
      + tags                 = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-us-east-1a"
        }
      + tags_all             = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-us-east-1a"
        }
      + vpc                  = true
    }

  # module.vpc.aws_internet_gateway.this[0] will be created
  + resource "aws_internet_gateway" "this" {
      + arn      = (known after apply)
      + id       = (known after apply)
      + owner_id = (known after apply)
      + tags     = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop"
        }
      + tags_all = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop"
        }
      + vpc_id   = (known after apply)
    }

  # module.vpc.aws_nat_gateway.this[0] will be created
  + resource "aws_nat_gateway" "this" {
      + allocation_id        = (known after apply)
      + association_id       = (known after apply)
      + connectivity_type    = "public"
      + id                   = (known after apply)
      + network_interface_id = (known after apply)
      + private_ip           = (known after apply)
      + public_ip            = (known after apply)
      + subnet_id            = (known after apply)
      + tags                 = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-us-east-1a"
        }
      + tags_all             = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-us-east-1a"
        }
    }

  # module.vpc.aws_route.private_nat_gateway[0] will be created
  + resource "aws_route" "private_nat_gateway" {
      + destination_cidr_block = "0.0.0.0/0"
      + id                     = (known after apply)
      + instance_id            = (known after apply)
      + instance_owner_id      = (known after apply)
      + nat_gateway_id         = (known after apply)
      + network_interface_id   = (known after apply)
      + origin                 = (known after apply)
      + route_table_id         = (known after apply)
      + state                  = (known after apply)

      + timeouts {
          + create = "5m"
        }
    }

  # module.vpc.aws_route.public_internet_gateway[0] will be created
  + resource "aws_route" "public_internet_gateway" {
      + destination_cidr_block = "0.0.0.0/0"
      + gateway_id             = (known after apply)
      + id                     = (known after apply)
      + instance_id            = (known after apply)
      + instance_owner_id      = (known after apply)
      + network_interface_id   = (known after apply)
      + origin                 = (known after apply)
      + route_table_id         = (known after apply)
      + state                  = (known after apply)

      + timeouts {
          + create = "5m"
        }
    }

  # module.vpc.aws_route_table.private[0] will be created
  + resource "aws_route_table" "private" {
      + arn              = (known after apply)
      + id               = (known after apply)
      + owner_id         = (known after apply)
      + propagating_vgws = (known after apply)
      + route            = (known after apply)
      + tags             = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-private"
        }
      + tags_all         = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-private"
        }
      + vpc_id           = (known after apply)
    }

  # module.vpc.aws_route_table.public[0] will be created
  + resource "aws_route_table" "public" {
      + arn              = (known after apply)
      + id               = (known after apply)
      + owner_id         = (known after apply)
      + propagating_vgws = (known after apply)
      + route            = (known after apply)
      + tags             = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-public"
        }
      + tags_all         = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop-public"
        }
      + vpc_id           = (known after apply)
    }

  # module.vpc.aws_route_table_association.private[0] will be created
  + resource "aws_route_table_association" "private" {
      + id             = (known after apply)
      + route_table_id = (known after apply)
      + subnet_id      = (known after apply)
    }

  # module.vpc.aws_route_table_association.private[1] will be created
  + resource "aws_route_table_association" "private" {
      + id             = (known after apply)
      + route_table_id = (known after apply)
      + subnet_id      = (known after apply)
    }

  # module.vpc.aws_route_table_association.private[2] will be created
  + resource "aws_route_table_association" "private" {
      + id             = (known after apply)
      + route_table_id = (known after apply)
      + subnet_id      = (known after apply)
    }

  # module.vpc.aws_route_table_association.public[0] will be created
  + resource "aws_route_table_association" "public" {
      + id             = (known after apply)
      + route_table_id = (known after apply)
      + subnet_id      = (known after apply)
    }

  # module.vpc.aws_route_table_association.public[1] will be created
  + resource "aws_route_table_association" "public" {
      + id             = (known after apply)
      + route_table_id = (known after apply)
      + subnet_id      = (known after apply)
    }

  # module.vpc.aws_route_table_association.public[2] will be created
  + resource "aws_route_table_association" "public" {
      + id             = (known after apply)
      + route_table_id = (known after apply)
      + subnet_id      = (known after apply)
    }

  # module.vpc.aws_subnet.private[0] will be created
  + resource "aws_subnet" "private" {
      + arn                                            = (known after apply)
      + assign_ipv6_address_on_creation                = false
      + availability_zone                              = "us-east-1a"
      + availability_zone_id                           = (known after apply)
      + cidr_block                                     = "10.0.10.0/24"
      + enable_dns64                                   = false
      + enable_resource_name_dns_a_record_on_launch    = false
      + enable_resource_name_dns_aaaa_record_on_launch = false
      + id                                             = (known after apply)
      + ipv6_cidr_block_association_id                 = (known after apply)
      + ipv6_native                                    = false
      + map_public_ip_on_launch                        = false
      + owner_id                                       = (known after apply)
      + private_dns_hostname_type_on_launch            = (known after apply)
      + tags                                           = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-private-us-east-1a"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/internal-elb"                         = "1"
        }
      + tags_all                                       = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-private-us-east-1a"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/internal-elb"                         = "1"
        }
      + vpc_id                                         = (known after apply)
    }

  # module.vpc.aws_subnet.private[1] will be created
  + resource "aws_subnet" "private" {
      + arn                                            = (known after apply)
      + assign_ipv6_address_on_creation                = false
      + availability_zone                              = "us-east-1b"
      + availability_zone_id                           = (known after apply)
      + cidr_block                                     = "10.0.11.0/24"
      + enable_dns64                                   = false
      + enable_resource_name_dns_a_record_on_launch    = false
      + enable_resource_name_dns_aaaa_record_on_launch = false
      + id                                             = (known after apply)
      + ipv6_cidr_block_association_id                 = (known after apply)
      + ipv6_native                                    = false
      + map_public_ip_on_launch                        = false
      + owner_id                                       = (known after apply)
      + private_dns_hostname_type_on_launch            = (known after apply)
      + tags                                           = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-private-us-east-1b"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/internal-elb"                         = "1"
        }
      + tags_all                                       = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-private-us-east-1b"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/internal-elb"                         = "1"
        }
      + vpc_id                                         = (known after apply)
    }

  # module.vpc.aws_subnet.private[2] will be created
  + resource "aws_subnet" "private" {
      + arn                                            = (known after apply)
      + assign_ipv6_address_on_creation                = false
      + availability_zone                              = "us-east-1c"
      + availability_zone_id                           = (known after apply)
      + cidr_block                                     = "10.0.12.0/24"
      + enable_dns64                                   = false
      + enable_resource_name_dns_a_record_on_launch    = false
      + enable_resource_name_dns_aaaa_record_on_launch = false
      + id                                             = (known after apply)
      + ipv6_cidr_block_association_id                 = (known after apply)
      + ipv6_native                                    = false
      + map_public_ip_on_launch                        = false
      + owner_id                                       = (known after apply)
      + private_dns_hostname_type_on_launch            = (known after apply)
      + tags                                           = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-private-us-east-1c"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/internal-elb"                         = "1"
        }
      + tags_all                                       = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-private-us-east-1c"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/internal-elb"                         = "1"
        }
      + vpc_id                                         = (known after apply)
    }

  # module.vpc.aws_subnet.public[0] will be created
  + resource "aws_subnet" "public" {
      + arn                                            = (known after apply)
      + assign_ipv6_address_on_creation                = false
      + availability_zone                              = "us-east-1a"
      + availability_zone_id                           = (known after apply)
      + cidr_block                                     = "10.0.0.0/24"
      + enable_dns64                                   = false
      + enable_resource_name_dns_a_record_on_launch    = false
      + enable_resource_name_dns_aaaa_record_on_launch = false
      + id                                             = (known after apply)
      + ipv6_cidr_block_association_id                 = (known after apply)
      + ipv6_native                                    = false
      + map_public_ip_on_launch                        = true
      + owner_id                                       = (known after apply)
      + private_dns_hostname_type_on_launch            = (known after apply)
      + tags                                           = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-public-us-east-1a"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/elb"                                  = "1"
        }
      + tags_all                                       = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-public-us-east-1a"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/elb"                                  = "1"
        }
      + vpc_id                                         = (known after apply)
    }

  # module.vpc.aws_subnet.public[1] will be created
  + resource "aws_subnet" "public" {
      + arn                                            = (known after apply)
      + assign_ipv6_address_on_creation                = false
      + availability_zone                              = "us-east-1b"
      + availability_zone_id                           = (known after apply)
      + cidr_block                                     = "10.0.1.0/24"
      + enable_dns64                                   = false
      + enable_resource_name_dns_a_record_on_launch    = false
      + enable_resource_name_dns_aaaa_record_on_launch = false
      + id                                             = (known after apply)
      + ipv6_cidr_block_association_id                 = (known after apply)
      + ipv6_native                                    = false
      + map_public_ip_on_launch                        = true
      + owner_id                                       = (known after apply)
      + private_dns_hostname_type_on_launch            = (known after apply)
      + tags                                           = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-public-us-east-1b"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/elb"                                  = "1"
        }
      + tags_all                                       = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-public-us-east-1b"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/elb"                                  = "1"
        }
      + vpc_id                                         = (known after apply)
    }

  # module.vpc.aws_subnet.public[2] will be created
  + resource "aws_subnet" "public" {
      + arn                                            = (known after apply)
      + assign_ipv6_address_on_creation                = false
      + availability_zone                              = "us-east-1c"
      + availability_zone_id                           = (known after apply)
      + cidr_block                                     = "10.0.2.0/24"
      + enable_dns64                                   = false
      + enable_resource_name_dns_a_record_on_launch    = false
      + enable_resource_name_dns_aaaa_record_on_launch = false
      + id                                             = (known after apply)
      + ipv6_cidr_block_association_id                 = (known after apply)
      + ipv6_native                                    = false
      + map_public_ip_on_launch                        = true
      + owner_id                                       = (known after apply)
      + private_dns_hostname_type_on_launch            = (known after apply)
      + tags                                           = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-public-us-east-1c"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/elb"                                  = "1"
        }
      + tags_all                                       = {
          + "Blueprint"                                               = "eks-blueprints-terraform-workshop"
          + "GithubRepo"                                              = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"                                                    = "eks-blueprints-terraform-workshop-public-us-east-1c"
          + "kubernetes.io/cluster/eks-blueprints-terraform-workshop" = "shared"
          + "kubernetes.io/role/elb"                                  = "1"
        }
      + vpc_id                                         = (known after apply)
    }

  # module.vpc.aws_vpc.this[0] will be created
  + resource "aws_vpc" "this" {
      + arn                                  = (known after apply)
      + cidr_block                           = "10.0.0.0/16"
      + default_network_acl_id               = (known after apply)
      + default_route_table_id               = (known after apply)
      + default_security_group_id            = (known after apply)
      + dhcp_options_id                      = (known after apply)
      + enable_classiclink                   = (known after apply)
      + enable_classiclink_dns_support       = (known after apply)
      + enable_dns_hostnames                 = true
      + enable_dns_support                   = true
      + enable_network_address_usage_metrics = (known after apply)
      + id                                   = (known after apply)
      + instance_tenancy                     = "default"
      + ipv6_association_id                  = (known after apply)
      + ipv6_cidr_block                      = (known after apply)
      + ipv6_cidr_block_network_border_group = (known after apply)
      + main_route_table_id                  = (known after apply)
      + owner_id                             = (known after apply)
      + tags                                 = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop"
        }
      + tags_all                             = {
          + "Blueprint"  = "eks-blueprints-terraform-workshop"
          + "GithubRepo" = "github.com/aws-ia/terraform-aws-eks-blueprints"
          + "Name"       = "eks-blueprints-terraform-workshop"
        }
    }

Plan: 23 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  + vpc_id = (known after apply)
```

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+locals {
+
+  name            = basename(path.cwd)
+  region          = data.aws_region.current.name
+  cluster_version = "1.24"
+
+  vpc_cidr = "10.0.0.0/16"
```
Comment (sujith-es) [inline locals.tf:7]: what does /16 mean?

File: locals.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  azs      = slice(data.aws_availability_zones.available.names, 0, 3)
+
+  node_group_name = "managed-ondemand"
+
+  tags = {
+    Blueprint  = local.name
+    GithubRepo = "github.com/aws-ia/terraform-aws-eks-blueprints"
```
Comment (sujith-es) [inline locals.tf:14]: Can I change this tag data and add new values here?

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  version = "3.16.0"
+
+  name = local.name
+  cidr = local.vpc_cidr
+
+  azs  = local.azs
+  public_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 10)]
+
+  enable_nat_gateway   = true
```
Comment (sujith-es) [inline main.tf:12]: what is NAT Gateway? Can we make this false?

File: main.tf [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  create_igw           = true
```
Comment (sujith-es) [inline main.tf:13]: What is this IGW?

File: main.tf [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  enable_dns_hostnames = true
+  single_nat_gateway   = true
+
+  # Manage so we can name
+  manage_default_network_acl    = true
```
Comment (sujith-es) [inline main.tf:18]: What is network ACL?

File: providers.tf [hunk 6]
```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = ">= 1.0.1"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.47"
```
Comment (sujith-es) [inline providers.tf:7]: Is this the latest version? Could you check what is the latest version?

File: providers.tf [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+      source  = "hashicorp/kubernetes"
+      version = ">= 2.10"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = ">= 2.4.1"
+    }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.14"
```
Comment (sujith-es) [inline providers.tf:19]: Check the latest version. Don't make any changes. Just check

File: data.tf [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# Find the user currently in use by AWS
+data "aws_caller_identity" "current" {}
+
+# Region in which to deploy the solution
+data "aws_region" "current" {}
```
Comment (sujith-es) [inline data.tf:5]: what does `data` mean in Terraform

**Review by sujith-es — CHANGES_REQUESTED** [review]

**Comment by DivyaMatt** [comment]
@sujith-es  request to re-review.

**Review by sujith-es — APPROVED** [review]
Looks good. Apply the changes.

**Pull request merged**